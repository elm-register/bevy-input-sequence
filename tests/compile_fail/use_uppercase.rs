use bevy_input_sequence::prelude::*;

fn main() {
    let _ = key!(a);
}
