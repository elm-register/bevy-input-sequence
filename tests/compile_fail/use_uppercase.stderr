error: Use uppercase key names
 --> tests/compile_fail/use_uppercase.rs:4:13
  |
4 |     let _ = key!(a);
  |             ^^^^^^^
  |
  = note: this error originates in the macro `key` (in Nightly builds, run with -Z macro-backtrace for more info)
